src/scripts/pokemons/CaughtPokemon.ts

```diff
@@ -49,4 +38,38 @@ class CaughtPokemon {
         plainJS = ko.toJS(this);
         return Save.filter(plainJS, keep);
     }
+
+    public evoTrack(reset = false){
```

Comment:
I don't know what to call this, taking any and all suggestions!

Reply:
`checkForEvolution()`?